python/cudf/cudf/core/column/string.py

```diff
@@ -543,6 +543,10 @@ def indices(self):
             self._indices = out_dev_arr
         return self._indices
 
+    @property
+    def _nbytes(self):
+        return self.children[0][-1]
```

Comment:
I'm out of the loop. What does `-1` entail here?

Reply:
String columns have two children, `children[1]` is a column of values (characters) in the string column. `children[0]` is the "offsets" column containing integer offsets into the first value for each string in the column. Further, `chlidren[0]` has an extra element at the end indicating the total number of bytes. This could equivalently be `children[1].size`. I've changed it to that.

Reply:
Thanks for explaining. I think `children[1].size` gives context, but is it technically correct? `size` reports number of elements, but are the number of elements and the number of bytes always the same?

Reply:
Yes - since `children[1]` is a non-nullable column always of type `INT8` (https://github.com/rapidsai/cudf/blob/branch-0.12/cpp/docs/TRANSITIONGUIDE.md#strings-support)